### PR TITLE
Rating Component

### PR DESCRIPTION
**Pull request by saicharan-gouru**
Please review my rating component.
Rating

### CODE REVIEW

**Review by OmKakatkar — APPROVED** [review]
Nice UI looks really clean.
Variable names seem to be meaningful and consistent throughout.
Looks good to merge

**Review by Rohitprasad83 — APPROVED** [review]
Code Looks Clean!

File: Components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                style="width: 100%; height: 300px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
```
Comment (Shailseen) [inline Components/Rating/rating.html:54]: Can you use external css instead of inline css ?

**Review by Shailseen — COMMENTED** [review]
All code looks fine commented part to be need to change.

**Pull request merged**